lca-cli/initmonitor/initmonitor.go

```diff
@@ -0,0 +1,107 @@
+package initmonitor
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/openshift-kni/lifecycle-agent/internal/ostreeclient"
+	"github.com/openshift-kni/lifecycle-agent/internal/reboot"
+	"github.com/openshift-kni/lifecycle-agent/lca-cli/ops"
+	rpmostreeclient "github.com/openshift-kni/lifecycle-agent/lca-cli/ostreeclient"
+
+	"github.com/go-logr/logr"
+)
+
+type InitMonitor struct {
+	scheme               *runtime.Scheme
+	log                  *logrus.Logger
+	hostCommandsExecutor ops.Execute
+	ops                  ops.Ops
+	component            string
+	rpmOstreeClient      *rpmostreeclient.Client
+	ostreeClient         ostreeclient.IClient
+}
+
+func NewInitMonitor(scheme *runtime.Scheme, log *logrus.Logger, hostCommandsExecutor ops.Execute, ops ops.Ops, component string) *InitMonitor {
+	return &InitMonitor{
+		scheme:               scheme,
+		log:                  log,
+		hostCommandsExecutor: hostCommandsExecutor,
+		ops:                  ops,
+		component:            component,
+		rpmOstreeClient:      rpmostreeclient.NewClient("initmonitor", hostCommandsExecutor),
+		ostreeClient:         ostreeclient.NewClient(hostCommandsExecutor, false),
+	}
+}
+
+func (m *InitMonitor) RunInitMonitor() error {
+	rollbackCfg, err := reboot.ReadIBUAutoRollbackConfigFile()
+	if err != nil {
+		if os.IsNotExist(err) {
+			return nil
```

Comment:
A log here as well, saying it will not run because the config did not exist

Reply:
I avoided that so that it wouldn't generate logs in the IBI scenario

Reply:
Logs are virtually free, if it helps troubleshooting in case the file doesn't exist because <some bug> in IBU, I think it's worth it

Reply:
If you don't mind the noise in the IBI journal logs, I can add them in, sure.

Reply:
If the log is not relevant in IBI then should not be generated, just creates confusion